index.js

```diff
@@ -0,0 +1,26 @@
+const hamburger = document.querySelector('.button1');
+const navMenu = document.querySelector('.orders-cont');
+const cross = document.querySelector('.x-button');
+const link = document.querySelectorAll('.nav-link');
+const section = document.querySelector('.section-2');
+
+let i = 0;
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  section.classList.toggle('active');
+});
```

Comment:
- [ ] Kindly add blur effect to the rest of parts(nav, work section) behind mobile menus to match it like the design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/173673950-1cc4c214-8ccd-4c69-b322-4548bff2dd2a.png) | ![image](https://user-images.githubusercontent.com/53402600/173674041-31685c78-9278-4d95-bfe7-ba0a8bea447f.png)|

Reply:
done